packages/react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -64,16 +62,8 @@ function useRenderedContent(
 
   const { rootDir, needsTheme } = getDir(props);
 
-  const { providerRef: parentProviderRef } = React.useContext(FocusRectsContext);
-  const focusRectsContext = React.useMemo<IFocusRectsContext>(
-    () => ({
-      providerRef: parentProviderRef ?? rootElement,
-    }),
-    [parentProviderRef, rootElement],
-  );
-
   let renderedContent = (
-    <FocusRectsProvider value={focusRectsContext}>
+    <FocusRectsProvider providerRef={rootElement}>
```

Comment:
FYI: The logic to inherit the parentProvideRef has been moved inside FocusRectsProvider